File: src/main/java/net/greemdev/core/util/ServerUtil.java
package net.greemdev.core.util;

import org.bukkit.Bukkit;
import org.bukkit.entity.Player;

import java.util.List;
import java.util.stream.Collectors;

public class ServerUtil {

    private ServerUtil() {}

    public static List<Player> getOnlineOps() {
        return Bukkit.getOnlinePlayers().stream().filter(p -> p.isOp() || p.hasPermission("*")).collect(Collectors.toList());
    }

    public static List<Player> getOnlineNormalPlayers() {
        return Bukkit.getOnlinePlayers().stream().filter(p -> !p.isOp()).collect(Collectors.toList());
    }

}

File: src/main/kotlin/net/greemdev/core/util/ServerUtil.kt
package net.greemdev.core.util

import org.bukkit.Bukkit
import org.bukkit.entity.Player
import java.util.stream.Collectors

public class ServerUtil {
    companion object {
        public fun getOnlineOps(): List<Player> {
            return Bukkit.getOnlinePlayers().stream().filter { p: Player ->
                p.isAdmin()
            }.collect(Collectors.toList())
        }

        public fun getOnlineNormalPlayers(): List<Player> {
            return Bukkit.getOnlinePlayers().stream().filter { p: Player ->
                !p.isAdmin()
            }.collect(Collectors.toList())
        }
    }
}